src/components/ui/modal/MyModal.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from './MyModal.module.css'
+const MyModal = (props) => {
```

Comment:
1. You can destructure props in function parameters and styles:
```suggestion
const MyModal = ({ visible, setVisible, children }) => {
    const { myModal, active, myModalContent } = styles;
```